index.js

```diff
@@ -16,6 +16,7 @@ const waitForUrl = async (url, MAX_TIMEOUT) => {
   for (let i = 0; i < iterations; i += 1) {
     try {
       await axios.get(url);
+      console.log('Url fetch completed, done!');
```

Comment:
Did you mean to leave this console log?

Reply:
Yes, I figured it'd be helpful to see whether the url was set, though maybe this is unneeded in the end